src/test/java/org/mskcc/cbio/oncokb/web/rest/RecaptchaAssessmentIT.java

```diff
@@ -0,0 +1,86 @@
+package org.mskcc.cbio.oncokb.web.rest;
+
+import org.junit.jupiter.api.*;
+import org.mskcc.cbio.oncokb.OncokbPublicApp;
+import org.mskcc.cbio.oncokb.config.application.ApplicationProperties;
+import org.mskcc.cbio.oncokb.config.application.RecaptchaProperties;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.mock.web.MockHttpServletRequest;
+
+import com.google.cloud.recaptchaenterprise.v1.RecaptchaEnterpriseServiceClient;
+import com.google.cloud.recaptchaenterprise.v1.RecaptchaEnterpriseServiceSettings;
+
+import javax.naming.ConfigurationException;
+import javax.xml.bind.ValidationException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@SpringBootTest(classes = OncokbPublicApp.class)
+@AutoConfigureMockMvc
+public class RecaptchaAssessmentIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RecaptchaAssessmentIT.class);
+
+    ApplicationProperties appProps;
+    RecaptchaProperties recaptchaProp;
+    RecaptchaEnterpriseServiceClient client;
+    private static String RECAPTCHA_TESTING_TOKEN = "faketoken";
+
+    @BeforeEach
+    public void setUp() throws IOException {
+        appProps = new ApplicationProperties();
+        recaptchaProp = new RecaptchaProperties();
+    }
+
+    @Test
+    public void testGetRecaptchaTokenWithToken() throws Exception {
+        recaptchaProp.setProjectId("symbolic-nation-320615");
+        recaptchaProp.setSiteKey("6LfAOe4jAAAAANjzxWQ8mKilcvk1QvLLohd7EV7F");
+        recaptchaProp.setThreshold((float) 0.5);
```

Comment:
@zhx828 These lines setting the `recaptchaProp` can be deleted because none of these tests use the client except for the last test where the `recaptchaProp` is deliberately left empty